src/layouts/EditPage/EditPage.tsx

```diff
@@ -70,7 +70,7 @@ export const EditPage = () => {
         pluginKey: "tableBubble",
       }),
       Table.configure({
-        resizable: true,
+        resizable: false,
```

Comment:
for clarification: why do we disable resizing of the table? Seems like we are using HTML now when saving the page, so users should be able to adjust the size of their tables to fit their requirements.

Reply:
hey good qn here, the users were confused about the resizability (doest translate to the actual preview)

Reply:
Makes sense yeah ok, thanks!